tutorials/darcy_thermo_mech/doc/content/workshop/intro/moose_multiphysics.md

```diff
@@ -6,7 +6,7 @@
 - Scientists are adept at creating applications in their domain
 - What about collaborating across research groups and/or disciplines?
 
-  - Head in the sand?
+  - Iterating between design teams?
```

Comment:
This isn't clear to me. This sounds more like something that would happen at a tech company

Reply:
Head in the sand was kind of sassy. People were finding their way around it, by iterating between the teams for TH, neutronics, fuel etc until everyone was happy enough